### ISSUE TITLE
Dispatching an event when user changed options  from configurator

### ISSUE THREAD

**Issue by felixhayashi** [almende/vis#1075]
Hi guys,

In addition to #940, would it be possible to also add an event for when the user changed the options with the help of the configurator? The point is, for all user interactions, vis allows me to react to the changes so I can automaticall persist them. For example, when a user zooms or dragsndrops something I can react to that. I also want to persist the graph options, but I do not know when they were changed.

If vis offered me an event-type for that I could do

```javascript
network.on("configChange", function() {
  persist(network.getOptionsFromConfigurator());
}
```

Thanks!
-Felix

**Comment by AlexDM0** [comment]
Hi Felix,

Makes sense, I'll see what I can do. You could also implement it yourself ;)

Regards

**Comment by AlexDM0** [comment]
Hi,

This has been added and will be released tomorrow.

Cheers

**Comment by felixhayashi** [comment]
Nice!! Many thanks @AlexDM0

**Comment by AlexDM0** [comment]
released in 4.5
